confgenerator/nginx.go

```diff
@@ -70,17 +70,19 @@ func (LoggingProcessorNginxAccess) Type() string {
 
 func (p LoggingProcessorNginxAccess) Components(tag string, uid string) []fluentbit.Component {
 	c := LoggingProcessorParseRegex{
-		// Sample line: ::1 - - [26/Aug/2021:16:49:43 +0000] "GET / HTTP/1.1" 200 10701 "-" "curl/7.64.0"
+		// Sample line (default config): ::1 - - [26/Aug/2021:16:49:43 +0000] "GET / HTTP/1.1" 200 10701 "-" "curl/7.64.0"
+		// Sample line (with latency in seconds): ::1 - - [26/Aug/2021:16:49:43 +0000] "GET / HTTP/1.1" 200 10701 "-" "curl/7.64.0" 1.002s
```

Comment:
The only documented log format is "combined" at http://nginx.org/en/docs/http/ngx_http_log_module.html and it does not contain a latency.

We shouldn't be encouraging users to define custom log formats. If users want a latency or response code metric, we should just expose those metrics in the metrics receiver.